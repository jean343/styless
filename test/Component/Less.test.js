import less from "less";

test('Less should still compile normally', async () => {
    const transpileLess = require("../../src/visitors/transpileLess").default;
    transpileLess("", "");

    const input = `
        @color: red;
        div {
            color: @color;
            darken: darken(@color, 20%);
        }
    `;
    const {css} = await less.render(input);
    expect(css).toMatchSnapshot();
});