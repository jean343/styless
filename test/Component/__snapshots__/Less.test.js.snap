// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Less should still compile normally 1`] = `
"div {
  color: red;
  darken: #990000;
}
"
`;
